services/admin-panels/docker-compose.aws.yml

```diff
@@ -3,3 +3,7 @@ services:
   adminpanels:
     dns: # Add this always for AWS, otherwise we get "No such image: " for docker services
       8.8.8.8
+    deploy:
+      placement:
+        constraints:
+          - node.role == manager
```

Comment:
question why the adminpanels must go on the manager? do they need the docker socket? if yes why is this not defined on the base docker-compose file instead of the AWS one?

Reply:
This is a good question. `admin-panels` are already set up to go on `manager`. @mrnicegyu11 do you know if there is any necessity in placing `admin-panels` on `manager` nodes?

Reply:
I dont think there is a reason anymore since we use docker configs. In the past there were some bind-mounted files so likely this is where the manager-constraint originally came from. I think it can be removed

Reply:
Moved to ops node